src/Apis/Types.ts

```diff
@@ -1,15 +1,18 @@
 export enum Paths {
     HOME = "/HOME",
     EVENTCALENDER = "/EVENTCALENDER",
+    EventInfo = "/EventInfo/:eventId",
 }
 
 export interface EventDay {
+    id: string;
```

Comment:
could be a better name convention like dayID or something more intuitive than id.